src/test/java/apiTests/TryApi.java

```diff
@@ -2,41 +2,52 @@
 
 import io.restassured.RestAssured;
 import io.restassured.response.ValidatableResponse;
-import json.data.MappperJSON;
+import json.issues.AddComment;
 import json.issues.Fields;
 import json.issues.Issue;
+import json.login.Login;
+import json.updatePriority.Priority;
+import json.updatePriority.SetId;
+import json.updatePriority.Update;
+import json.updatePriority.UpdateFields;
 import org.json.simple.JSONArray;
 import org.json.simple.JSONObject;
 import org.testng.Assert;
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeSuite;
 import org.testng.annotations.Test;
-import utils.SampleMapper;
+
+import java.util.Set;
 
 import static io.restassured.RestAssured.given;
 
 public class TryApi {
-    String username = "webinar5";
-    String password = "webinar5";
     String sessionId;
     String issueId;
     String commentId;
+    String set;
+
+    public static Login login;
+
 
     @BeforeSuite
     public void setupMethod(){
         RestAssured.baseURI = "http://jira.hillel.it";
         RestAssured.port = 8080;
-        JSONObject login = new JSONObject();
-        login.put("username",username);
-        login.put("password",password);
+
+        login = new Login();
+        login.setLogin("webinar5");
```

Comment:
Can this setters be replaced with constructor?
 new Login("webinar5", "webinar5");